client/state/action-types.js

```diff
@@ -29,7 +29,11 @@ export const ANALYTICS_PAGE_VIEW_RECORD = 'ANALYTICS_PAGE_VIEW_RECORD';
 export const ANALYTICS_STAT_BUMP = 'ANALYTICS_STAT_BUMP';
 export const AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST = 'AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST';
 export const AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE = 'AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE';
+export const AUTOMATED_TRANSFER_INITIATE = 'AUTOMATED_TRANSFER_INITIATE';
```

Comment:
@seear is working on that, but I think he and the rest of the team are AFK until January. (cc @folletto)